### ISSUE TITLE
"This application requires a Java Runtime Environment"

### ISSUE THREAD

**Issue by VannyBuns** [Ninjiteu/N64SaveConverter#3]
Getting this error when launching the program when I already have java

**Comment by Ninjiteu** [comment]
This application is written in java and requires java (JRE) to be installed on your computer. https://www.java.com/en/

**Comment by VannyBuns** [comment]
Yes I'm aware, I already have it. Reinstalled it, says the same thing.

**Comment by Ninjiteu** [comment]
Maybe check your environment variables and make sure your java installation bin directory is in your PATH variable.

**Comment by Ninjiteu** [comment]
You can open up a terminal and type in "java --version" and check to see if that works for you. It should return your java version info

**Comment by ToxBox-G** [comment]
Please provide instructions on how to install Java for this version. Every single java version wont work with this at all.

**Comment by ToxBox-G** [comment]
If anybody is looking for a solution, go to this repo. it works ASAP
https://github.com/drehren/ra_mp64_srm_convert

**Comment by Ninjiteu** [comment]
The issue has been resolved. Please download the N64SaveConverterGui.exe and download Java JDK 17.0.5 and set your environment variables JAVA_HOME/PATH. 

EASIEST SOLUTION: If that still doesn't work for you, just download N64SaveConverterGui.zip. It comes with the jdk bundled. You only need to extract the zip and run the exe. It should work fine without doing any additional work of setting any environment variables.

Hope that helps

**Comment by ToxBox-G** [comment]
Thank you for the quick reply, I appreciate it. I found this through multiple google keyword searches related to dex drive to modern emulator saves and this was brought up at few times. I reinstalled Java JDK 17.0.5 at few moments ago and the program still won't work but the zip work right away. So I recommend removing exe from the releases list cuz you may bump to another person like me looking for the solution. Plus make sure to put up the info of having to use zip file at top region of the readme so newbies on n64 emulation scene can have easier time running it without any stresses cuz dex drive is a outdated save file type.
